bash.yaml

```diff
@@ -61,6 +61,19 @@ subpackages:
           mkdir -p "${{targets.subpkgdir}}"/usr/lib/bash
           mv "${{targets.destdir}}"/usr/lib/bash/Makefile* "${{targets.subpkgdir}}"/usr/lib/bash/
 
+  - name: bash-binsh
+    dependencies:
+      runtime:
+        - bash
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/bin
+          ln -s bash "${{targets.subpkgdir}}"/bin/sh
```

Comment:
I sort of expect this PR to also change something in busybox which tries to install /bin/sh symlink with a scriplet.

Also apko has busybox profile to install said /bin/sh symlink too. Have you build an image locally with this package, and how does that interact with apko's busybox behaviour?

Reply:
I really would expect apko to work, as bash would not be the first thing that busybox has an applet for that is installable.

Ie, you have images with busybox and coreutils.

Reply:
There is magic in apko to read and use `/etc/busybox-paths.d/busybox` thing

Also i see that removal doesn't work correctly for dash already, so can take fixing that as a separate wishlist bug - https://github.com/wolfi-dev/os/issues/17605